src/utils/string/StringUtils.js

```diff
@@ -398,6 +398,11 @@ const onOpenLink = (url) => {
   });
 };
 
+const isValidUrl = (url) => {
+  const urlPattern = /^(https?:\/\/)?([\da-zA-Z.-]+)\.([a-zA-Z]{2,6})([/\w.-]*)*\/?$/;
+  return urlPattern.test(url);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The regular expression used for URL validation is not comprehensive. It does not account for URLs with query parameters, fragment identifiers, or authentication information. This could lead to valid URLs being incorrectly marked as invalid. Consider using a more robust method for URL validation, such as the built-in URL constructor in JavaScript.

```diff
- const urlPattern = /^(https?:\/\/)?([\da-zA-Z.-]+)\.([a-zA-Z]{2,6})([/\w.-]*)*\/?$/;
- return urlPattern.test(url);
+ try {
+   new URL(url);
+   return true;
+ } catch (_) {
+   return false;
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->